app/assets/javascripts/controllers/ems_infra_dashboard/utilization_trend_chart_controller.js

```diff
@@ -1,12 +1,12 @@
-angular.module( 'patternfly.charts' ).controller('utilizationTrendChartController', ['$q', 'providerId', 'chartsMixin', '$http', 'miqService', function($q, providerId, chartsMixin, $http, miqService) {
+angular.module( 'patternfly.charts' ).controller('utilizationTrendChartController', ['$q', 'providerDashboard', 'providerId', 'chartsMixin', '$http', 'miqService', function($q, providerDashboard, providerId, chartsMixin, $http, miqService) {
```

Comment:
Where exactly does the `providerDashboard` here come from?

Reply:
I'm asking, since this change leads to an Angular error:

```
Error: [$injector:unpr] Unknown provider: providerDashboardProvider
...
```

when navigating to Infra Providers -> [infra provider] -> Dashboard

Reply:
> Where exactly does the providerDashboard here come from?

Ops, my bad here, I shouldn't touch this file. opening a fix for this